cves/kernel/CVE-2021-31440.yml

```diff
@@ -221,7 +229,7 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
+  name: bpf
```

Comment:
Investigation Comment,

What does bpf mean? I think a description of the subsystem should be in the note section.

Reply:
I give this 3 upvotes.